com.unity.multiplayer.mlapi/Editor/MLAPIProfiler.cs

```diff
@@ -8,7 +8,7 @@ namespace UnityEditor
 {
     public class MLAPIProfiler : EditorWindow
     {
-#if !UNITY_2020_2_OR_NEWER
```

Comment:
At this point, should this just be removed as a whole? Since we were keeping the MLAPIProfiler for version under 2020.2 but 2019.4+. Or for now we keep it like this and remove it whole for 1.0?

Reply:
we aren't fully decided to make 2020.3 LTS baseline version requirement permanent yet. we might consider jumping back to 2019.4 LTS if we get some fixes around ILPP issues we're having — so it's still TBD, that's why I'm still keeping pre-2020.3 blocks around :)